src/mds/MDLog.cc

```diff
@@ -75,6 +75,8 @@ void MDLog::create_logger()
   plb.add_u64(l_mdl_rdpos, "rdpos", "Journaler  read position");
   plb.add_u64(l_mdl_jlat, "jlat", "Journaler flush latency");
 
+  plb.add_u64_counter(l_mdl_replay, "replay", "Events replayed");
```

Comment:
I'm not sure "replay" is descriptive enough; why not "events_replayed" or "replayed" (past tense) or something?